packages/docusaurus-theme-common/src/utils/ThemeClassnames.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const ThemeClassNames = {
+  page: {
+    blogListPage: 'blog-list-page',
+    blogPostPage: 'blog-post-page',
+    blogTagsListPage: 'blog-tags-list-page',
+    blogTagsPostPage: 'blog-tags-post-page',
+    docPage: 'doc-page',
+    mdxPage: 'mdx-page',
+  },
+  wrapper: {
+    blogPage: 'main-docs-wrapper',
+    docPage: 'blog-wrapper',
```

Comment:
```suggestion
    blogPage: 'blog-wrapper',
    docPage: 'main-docs-wrapper',
```

Looks interverted here

Reply:
also what about "blogWrapper" and "docsWrapper"?

Reply:
Woops, I must be tripping, fixing it right now!